packages/web3torrent/src/components/torrent-info/peer-speed-info/PeerSpeedInfo.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import prettier from 'prettier-bytes';
+import './PeerSpeedInfo.scss';
+import {TorrentUI, IdleStatuses} from '../../../types';
+
+export type PeerSpeedInfoProps = {torrent: TorrentUI};
+
+const PeerSpeedInfo: React.FC<PeerSpeedInfoProps> = ({torrent}: PeerSpeedInfoProps) =>
+  !IdleStatuses.includes(torrent.status) && (
+    <section className="peerSpeedInfo">
+      {!torrent.originalSeed && torrent.parsedTimeRemaining && torrent.parsedTimeRemaining + '. '}
```

Comment:
Might be just me, but I find this hard to read.

Reply:
Yeah, you are right, I'll modify that

Reply:
https://github.com/statechannels/monorepo/pull/1754/commits/fd1d498ce982e73e73956718ca2ff65377536e5a